src/components/OnBoarding/OnBoardingPage.js

```diff
@@ -69,32 +79,64 @@ class OnBoardingPage extends Component {
     return (
       <View style={{
         backgroundColor: '#a6aeae',
-        width: 7,
-        height: 7,
-        borderRadius: 4,
-        marginLeft: 6,
-        marginRight: 6,
-        marginTop: 3,
-        marginBottom: 3,
+        width: dimensions.fontWeight * 7,
+        height: dimensions.fontWeight * 7,
+        borderRadius: dimensions.fontWeight * 4,
+        marginLeft: dimensions.widthWeight * 6,
+        marginRight: dimensions.widthWeight * 6,
+        marginTop: dimensions.heightWeight * 3,
       }} />
     );
   }
 
+  controlScroll(e, state) {
```

Comment:
e is event,
This is original case of onMomentumScrollEnd for Scrollview,
for get state, e is needed maybe..